CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 * The Custom::ECSService custom resource now waits for newly created ECS services to stabilize [#878](https://github.com/remind101/empire/pull/878)
 * The CloudFormation backend now uses the Custom::ECSService resource instead of AWS::ECS::Service, by default [#877](https://github.com/remind101/empire/pull/877)
 * The database schema version is now checked at boot, as well as in the http health checks. [#893](https://github.com/remind101/empire/pull/893)
+* The log level within empire can now be configured when starting the service. [#929]((https://github.com/remind101/empire/issues/929)
```

Comment:
Do you wanna merge https://github.com/remind101/empire/pull/930 first then rebase this?